### PR TITLE
Handle edge case where an authorizedUser has multiple verification methods

### PR DESCRIPTION
**Pull request by Robert-H-Leonard**

### CODE REVIEW

File: test/DidRegistryVerificationMethodTest.sol [hunk 1]
```diff
@@ -58,13 +58,13 @@ contract DidRegistryVerificationMethodTest is DidRegistryTest {
 
     function test_authorized_user_should_remove_verification_method() public {
         address user = vm.addr(1);
-        address nonAuthorizedUser = vm.addr(2);
+        address authorizedUser = vm.addr(2);
```
Comment (rado0x54) [inline test/DidRegistryVerificationMethodTest.sol:61]: I assume this was just an unrelated typo?

**Review by rado0x54 — APPROVED** [review]
